doc/user_guide/marks/image.rst

```diff
@@ -43,3 +43,104 @@ Scatter Plot with Image Marks
     )
 
     alt.Chart(source).mark_image(width=50, height=50).encode(x="x", y="y", url="img")
+
+Show Image Marks with Selection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This example demonstrates how to display image marks with drag selection. We create two charts:
+one with point marks and the other with image marks, applying the selection filter only to the latter.
+By combining these two charts, we can achieve the desired result.
+
+.. altair-plot::
+    :hide-code:
+    :div_class: properties-example
+
+    import altair as alt
+    import pandas as pd
+
+    source = pd.DataFrame.from_records(
+        [{'a': 1, 'b': 1, 'image': 'https://altair-viz.github.io/_static/altair-logo-light.png'},
+        {'a': 2, 'b': 2, 'image': 'https://avatars.githubusercontent.com/u/11796929?s=200&v=4'},
+        {'a': 3, 'b': 3, 'image': ''}]
+    )
+
+    brush = alt.selection_interval()
+    point = alt.Chart(source, width=200, height=200).mark_circle(size=100).encode(
+        x='a',
+        y='b',
+    ).add_params(
+        brush
+    )
+
+    img = alt.Chart(source).mark_image(width=50, height=75).encode(
+        x='a',
+        y='b',
+        url='image'
+    ).transform_filter(
+        brush
+    )
+
+    img_faceted = alt.Chart(source, width=50, height=75).mark_image().encode(
+        url='image'
+    ).facet(
+        alt.Facet('image', title='', header=alt.Header(labelFontSize=0))
+    ).transform_filter(
+        brush
+    )
+
+
+    two_layered_chart = point + img
+    faceted_chart = point | img_faceted
+
+    alt.hconcat(
+        two_layered_chart.properties(title="Two Layered Chart"),
+        faceted_chart.properties(title="Faceted Chart"),
+    ).configure_title(
+        fontSize=10,
+        anchor='start'
+    )
+
+In the layered chart, images may overlap one other, using the faceted chart instead can
+avoid this issue.
+
+If you're looking to learn how to create an image tooltip that displays an image when hovering over a point, 
```

Comment:
Can you include the Image Tooltip example directly on this page under its own heading and remove the gallery example? I think this is a natural place for it and we have so many different gallery examples that it is sometimes hard to find what we are looking for there.

Reply:
Add Image Tooltip section
![image](https://github.com/altair-viz/altair/assets/75072960/ac162355-881e-472f-b5e9-9af5309b27d7)